R/pkg/R/context.R

```diff
@@ -437,3 +437,33 @@ setCheckpointDir <- function(directory) {
   sc <- getSparkContext()
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(directory))))
 }
+
+#' Total number of CPU cores of all executors registered in the cluster at the moment
+#' and potentially available to jobs submitted via the Spark context.
+#'
```

Comment:
@MaxGekk, is `The number reflects current status of the cluster and can change in the future` intentionally taken out here or a mistake?

Reply:
and `This method is experimental, and its behavior can be changed in the next releases.` too.